lib/ups/services.rb

```diff
@@ -1,21 +1,39 @@
 module UPS
   SERVICES = {
```

Comment:
@caseyli was unable to have 2 constants in the same module. This seemed to work.